protocol/streaming/full_node_streaming_manager.go

```diff
@@ -362,6 +362,34 @@ func (sm *FullNodeStreamingManagerImpl) SendOrderbookFillUpdates(
 	sm.AddUpdatesToCache(updatesByClobPairId, uint32(len(orderbookFills)))
 }
 
+// SendTakerOrderStatus sends out a taker order and its status to the full node streaming service.
+func (sm *FullNodeStreamingManagerImpl) SendTakerOrderStatus(
+	streamTakerOrder clobtypes.StreamTakerOrder,
+	blockHeight uint32,
+	execMode sdk.ExecMode,
+) {
+	clobPairId := uint32(0)
+	if liqOrder := streamTakerOrder.GetLiquidationOrder(); liqOrder != nil {
+		clobPairId = liqOrder.ClobPairId
+	}
+	if takerOrder := streamTakerOrder.GetOrder(); takerOrder != nil {
+		clobPairId = takerOrder.OrderId.ClobPairId
+	}
+
+	sm.AddUpdatesToCache(
+		map[uint32][]clobtypes.StreamUpdate{
+			clobPairId: {
+				{
+					UpdateMessage: &clobtypes.StreamUpdate_TakerOrder{
+						TakerOrder: &streamTakerOrder,
+					},
```

Comment:
for my understanding - do we need to send any fill amounts for taker orders?

Reply:
It should have the fill amounts in a following full node stream update if you are subscribed to the correct clob pair id!